python/src/truv.py

```diff
@@ -0,0 +1,221 @@
+import logging
```

Comment:
why renaming from client to truv?

Reply:
Just the same name as for the other languages. 
And it's shorter ;)